uwsgi/admin.wsgi

```diff
@@ -34,6 +34,10 @@ DOMAIN_ALLOWLIST = {
         "Widevine",
         "Widevine-L1",
     ),
+    "www.google.com": (
```

Comment:
This is a bit of a gaping hole in that Balrog would allow anything accessible through https://www.google.com to be served as a Widevine plugin... Is there any we can get a more scoped down domain for the mirrors? Or would that just end up with the same problem of things getting blocked?

Reply:
I could see how that might get abused, but part of the point is to use a domain that is unlikely to be blocked. The URL does have a set form though, perhaps I could extend the allowlisting to support partial URLs, e.g. https://www.google.com/dl/release2/chrome_component

Reply:
This seems to be the only place that uses the result directly, so I could change the format a bit to provide both domains and URLs:
https://github.com/mozilla-releng/balrog/blob/b2d6bd30a05cc430e6067c7375797234267ec3ca/src/auslib/AUS.py#L38

If it doesn't match a domain, it can check to see if the URL matches one of the starting fragments instead. Change DOMAIN_ALLOWLIST to something like
```
DOMAIN_ALLOWLIST = {
  "domains": {
    // ... existing list
  },
  "urlFragments": {
    "https://www.google.com/dl/release2/chrome_component": (
        "Widevine",
        "Widevine-L1",
    )
  },
}
```

Reply:
I did a more advanced version of the latter and updated the PR. Now it can take in an optional fullmatch regex for the path to ensure more generic domains don't get abused. The regex for www.google.com should only allow us to access files from the chrome component update service.